tensorflow/python/ops/array_ops_test.py

```diff
@@ -143,6 +143,32 @@ def testLowerBoundInputsWrongRank(self):
     ):
       gen_array_ops.lower_bound(arg0, arg1)
 
+  def testWrapPadOp(self):
+    # Op implemented in Python so test is found here.
+    t = array_ops.constant([[1, 2, 3], [4, 5, 6]])
+    paddings = array_ops.constant([[1, 1,], [2, 2]])
+    self.assertAllEqual(
+      array_ops._wrap_pad(t, paddings), array_ops.constant([
+                                      [5, 6, 4, 5, 6, 4, 5],
+                                      [2, 3, 1, 2, 3, 1, 2],
+                                      [5, 6, 4, 5, 6, 4, 5],
+                                      [2, 3, 1, 2, 3, 1, 2]] )
+    )
+
+  def testWrapPadConcreteOp(self):
+    wrap_pad_concrete = array_ops._wrap_pad.get_concrete_function(
```

Comment:
`_wrap_pad` is not a `tf.function`, so we get the following error:
```
Traceback (most recent call last):
  File "tensorflow/python/ops/array_ops_test.py", line 167, in testWrapPadConcreteOp
    wrap_pad_concrete = array_ops._wrap_pad.get_concrete_function(
AttributeError: 'function' object has no attribute 'get_concrete_function'
```
You need to do something like
```
def_function.function(array_ops._wrap_pad).get_concrete_function(...)
```

Reply:
done